docs/design-proposals/cloud-edge-communication/cloud-edge-communication-framework.md

```diff
@@ -11,18 +11,255 @@ Because the cloud and edge is located in the network boundary of different secur
 However, there may be many problems when VPN is used to directly connect the components/services of different security levels:
 
 * It is too risky to directly expose the central cloud management network/IP to users
-
 * IP needs to be planned globally in advance to ensure that the management components/service IP of the central cloud and each edge site do not conflict
-
 * Network changes can be difficult
 
 This proposal aims to outline a design for cloud-edge communication framework/service to solve the above problems
 
 ## Goals
 
 * Support communication access capability of cloud-edge basic management components
-
 * Support independent IP addressing and IP address convergence of edge management components/services
-
 * Minimize attack surface, zero trust security
 
+## Proposal
+
+The following two solutions do not conflict with each other. The former can converge IP addresses to resolve IP address conflicts. However, the communication needs to be first sent to the gateway and then to the destination service. The latter cannot converge IP addresses but can resolve IP address conflicts. The latter provides end-to-end communication.
```

Comment:
Please add a reason why two options that are not conflicting with each other are presented? are we supposed to pick one? or are we suppose to find away to merge them? The statement of the intention would help guide reading each of the proposals.

Reply:
We want to choose proposal B, the description here will be deleted for no ambiguity, thx